core-developers/experts.rst

```diff
@@ -244,7 +244,7 @@ tracemalloc           vstinner
 tty                   Yhg1s*
 turtle                gregorlingl^, willingc
 types                 1st1
-typing                gvanrossum, Fidget-Spinner, JelleZijlstra*
+typing                gvanrossum, Fidget-Spinner, JelleZijlstra*, AlexWaygood*,
```

Comment:
I think the trailing comma isnt supposed to be there.

Reply:
Oops
```suggestion
typing                gvanrossum, Fidget-Spinner, JelleZijlstra*, AlexWaygood*
```